config/settings.yml

```diff
@@ -27,6 +27,11 @@ github_notification_monitor:
   repo_names: []
 merge_target_titler:
   enabled_repos: []
+path_based_labeler:
+  enabled_repos: []
+  rules: []
+    # - pattern:
+    #   label:
```

Comment:
I think I'll still have to tweak [the check](https://github.com/ManageIQ/miq_bot/blob/637e49616839c5765d9c2f050c43338afc396f04/app/models/repo.rb#L72-L74) to account for that different format but that shouldn't be terribly difficult. Alrighty.